Apps/W1/HybridGP/app/src/Migration/Items/GPItemMigrator.codeunit.al

```diff
@@ -370,4 +395,59 @@ codeunit 4019 "GP Item Migrator"
                 exit(CostingMethodOption::Standard);
         end;
     end;
+
+    procedure MigrateItemClassesIfNeeded(GPItem: Record "GP Item"; ItemDataMigrationFacade: Codeunit "Item Data Migration Facade")
+    var
+        GPItemLocation: Record "GP Item Location";
+        InventoryPostingGroup: Record "Inventory Posting Group";
+        GPIV00101: Record "GP IV00101";
+        GPIV40400: Record "GP IV40400";
+        HelperFunctions: Codeunit "Helper Functions";
+        PostingGroupCode: Code[20];
+        AccountNumber: Code[20];
+    begin
+        if DefaultAccountNumber = '' then
+            DefaultAccountNumber := HelperFunctions.GetPostingAccountNumber(InventoryAccountName);
+
+        AccountNumber := DefaultAccountNumber;
+
+        // Default Posting Group (GP)
+        PostingGroupCode := CopyStr(DefaultPostingGroupCodeTxt, 1, 20);
+        if not InventoryPostingGroup.Get(PostingGroupCode) then begin
+            ItemDataMigrationFacade.CreateInventoryPostingSetupIfNeeded(PostingGroupCode, CopyStr(DefaultPostingGroupDescriptionTxt, 1, 50), '');
```

Comment:
We should use MaxStrLen here instead of hardcoding. The same for below lines

Reply:
Good tip, I'll use this from now on.